test/Microsoft.Restier.EntityFramework.Tests/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="EntityFramework" version="6.1.1" targetFramework="net45" />
+  <package id="Microsoft.OData.Edm" version="6.9.0" targetFramework="net45" />
```

Comment:
Not for this change: We shall update the ODataLib dependency to a newer version.